src/pydata_sphinx_theme/theme/pydata_sphinx_theme/layout.html

```diff
@@ -61,7 +61,7 @@
   {# the scroll to top button #}
   <button type="button" class="btn rounded-pill" id="pst-back-to-top">
     <i class="fa-solid fa-arrow-up"></i>
-    {{ _("Back to top") }}
+    {{- _("Back to top") -}}
```

Comment:
Mostly out of curiosity: why are these new `-` needed?

Reply:
It removes the whitespace around the text.

I'm not 100% sure they're needed. When I added the underline on hover, it didn't look good because of the underlined space (which may have been a space character, I'm not 100% sure) between the up arrow and the text. 

Then I added the minus signs. But then the text was too close to the arrow. So then I added some margin to the arrow. But after I added the margin, I never checked if the minus signs could be removed, but my hunch is that if you take the minus signs back out, then there will be too much space between the up arrow and the text. Although the minus side on the right side probably isn't needed.